modules/dynamic-apiserver/pkg/cli/install_operation.go

```diff
@@ -58,6 +58,10 @@ const (
 	installReportErrors                 = "report_errors"
 	installSentryDSNParam               = "sentry_dsn"
 	installDeploymentId                 = "deployment_id"
+	installUseCustomDockerRegistryParam = "use_custom_docker_registry"
```

Comment:
I suggest to avoid "custom". Instead of "use_custom_docker_registry" -> "use_docker_registry" etc.

Reply:
Done